src/components/Shared/BetForm.tsx

```diff
@@ -48,17 +57,29 @@ const BetForm = (props: GamePanelProps) => {
         <>
             <p>
                 Tipo de apuesta:
-                <select className="full-width">
-                    <option selected disabled={true}>Cl&aacute;sica (3 puntos)</option>
+                <select className="full-width" value={betType} onChange={changeBetType}>
+                    <option value="CLASSIC">Cl&aacute;sica (3 puntos)</option>
+                    <option value="DOUBLE_CHANCE">Doble oportunidad (1 punto)</option>
                 </select>
             </p>
             <p>
                 Tu resultado:
-                <select className="full-width" value={betResult} onChange={(e) => changeBetResult(e)}>
-                    {!props.bet && <option value={""} disabled={true}>Seleccione una opci&oacute;n</option>}
-                    <option value="HOME_TEAM_WON">Gana {props.game.homeTeam.name} (Local)</option>
-                    <option value="AWAY_TEAM_WON">Gana {props.game.awayTeam.name} (Visitante)</option>
-                    <option value="DRAW">Empate</option>
+                <select className="full-width" value={betResult} onChange={changeBetResult}>
+                    {!props.bet && <option value="" disabled={true}>Seleccione una opci&oacute;n</option>}
```

Comment:
Let's show the "select an option" label if betResult is blank (this happens the first time you make a bet or if you change the bet type).

```suggestion
                    {!betResult && <option value="" disabled={true}>Seleccione una opci&oacute;n</option>}
```